### PR TITLE
Api narrowed down searching algorithm

### PR DESCRIPTION
**Pull request by nsirimo**

### CODE REVIEW

**Comment by clarios6** [comment]
Your bestNum array only works once!

File: server/server.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ var mongoose = require('mongoose');
 var app = express();
 var Subdistrict = require('./Models/Subdistrict');
 var Parkinglot = require('./Models/Parkinglot');
+var bestNum = [];
```
Comment (clarios6) [inline server/server.js]: this line only works the first time, this is out of scope!

**Comment by nsirimo** [comment]
@clarios6 I have made these changes.

**Comment by clarios6** [comment]
I have tested this with multiple addresses and it works as intended.
